js/utils.js

```diff
@@ -994,6 +994,34 @@ const CombodoJSConsole = {
 	}
 }
 
+/**
+ * Helper to reflect ongoing JS activity to other processes like BeHat
+ * @api
+ * @since 3.0.4 3.1.1 3.2.0 N°6765
+ */
+const CombodoJsActivity = {
+	BODY_DATAATTR_NAME_READY: "data-ready-scripts",
+
+	/**
+	 * Counter so that we set the flag as done only on the last call
+	 * @type number
+	 */
+	OngoingScriptsCount: 0,
+
+	AddOngoingScript: function() {
+		this.OngoingScriptsCount++;
+		$("body").attr(this.BODY_DATAATTR_NAME_READY, "start");
+	},
+
+	RemoveOngoingScript: function() {
+		this.OngoingScriptsCount--;
+
+		if (this.OngoingScriptsCount < 1) {
```

Comment:
```suggestion
	iOngoingScriptsCount: 0,

	AddOngoingScript: function() {
		this.iOngoingScriptsCount++;
		$("body").attr(this.BODY_DATAATTR_NAME_READY, "start");
	},

	RemoveOngoingScript: function() {
		this.iOngoingScriptsCount--;

		if (this.iOngoingScriptsCount < 1) {
```

To be consistent with other helpers, then ok to merge.

Reply:
Done